nhais/sequence/tests/test_dynamo_sequence.py

```diff
@@ -0,0 +1,78 @@
+"""Module to test dynamo sequence functionality."""
+import unittest
+
+import aioboto3
+from sequence import dynamo_sequence
+from utilities import test_utilities
+from utilities import config
+
+
+class TestDynamoSequence(unittest.TestCase):
```

Comment:
Since this test requires Dynamo to be running it should be an integration test rather than a unit tests.

Is there anything we can unit test?

Reply:
"component test" rather than "integration test"

Reply:
file renamed